data/PA/incentives.json

```diff
@@ -0,0 +1,1023 @@
+    },
+    "start_date": "2021",
+    "end_date": "2026"
+  },
+  {
+    "id": "PA-25",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "heat_pump_clothes_dryer",
+    "program": "pa_firstEnergy_residentialProductsRebateProgram",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 75,
+      "maximum": 75
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$75 rebate for ENERGY STAR certified clothes dryer for income-qualified residents."
+    },
+    "start_date": "2021",
+    "end_date": "2026",
+    "low_income": "pa-first-energy"
+  },
+  {
+    "id": "PA-26",
+    "authority_type": "other",
+    "authority": "pa-first-energy",
+    "eligible_geo_group": "pa-group-first-energy",
+    "payment_methods": [
+      "assistance_program"
+    ],
+    "item": "weatherization",
+    "program": "pa_firstEnergy_wARMProgram",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Free energy audit and weatherization services for income-qualified residents."
+    },
+    "low_income": "pa-first-energy-warm"
+  },
+  {
+    "id": "PA-27",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 350
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $350 rebate for an in-home audit for electric heating and/or central A/C."
+    }
+  },
+  {
+    "id": "PA-28",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "percent",
+      "number": 1
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Free virtual home energy assessment. Assessment comes with a free energy savings kit mailed to home."
+    }
+  },
+  {
+    "id": "PA-29",
+    "authority_type": "utility",
+    "authority": "pa-ppl-electric-utilities-corp",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "other",
+    "program": "pa_pPLElectric_products&Rebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 300
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Up to $300 rebate for central air conditioner. Rebate value depends on equipment efficiency."
+    },
+    "bonus_available": true
+  },
+  {
+    "id": "PA-30",
```

Comment:
This isn't something to solve in this PR, but this is not ideal. PA-30 through PA-33 will show up in the frontend as four cards:

- "75% up to $200 for weatherization"
- "75% up to $500 for weatherization"
- "75% up to $200 for weatherization"
- "75% up to $500 for weatherization"

...all with the same program name.

It will look terrible and the actual distinctions (basement vs. attic, electric vs. non-electric heat) will only be in the fine print.

I thought one possible quick fix would be to make the electric vs. non-electric heat pairs mutually exclusive, since your house is either electric-heated or not. But that would be equivalent to completely omitting one of them, since the calculation logic would find either both or neither ineligible on the basis of other factors. (I think we talked about this problem on one of your other PRs?)

Really don't know what to do here! I've been thinking about adding a concept of sub-item (which would allow a distinction between attic and basement insulation), but that doesn't help answer what this should look like in the frontend.

Reply:
this has come up a couple times in QA sessions as well and we haven't been totally sure how to handle it. I think we have talked about making electric vs non electric mutually exclusive, but I agree that isn't the right call here (although if we coded these differently in the API we could show different incentives in PEP where we do know a user's heating source).

I think the easiest fix right now is to combine these 4 into 2 incentives, 1 for electric and 1 for non electric, and add more details to the description. something like:
- PA-30: Rebate for 75% of costs up to $1000 for insulation in an electric heated house. Up to $500 for attic insulation and $500 for basement wall insulation.
- PA-31: Rebate for 75% of costs up to $400 for insulation (central AC, non-electric heat). Up to $200 for attic insulation and $200 for basement wall insulation.
- Delete PA-32 and PA-33

thoughts?

Reply:
Yeah, I like that. I think the description should start with the electric/non-electric distinction and have the attic/basement part after.